plyer/platforms/android/qrbarcodereader.py

```diff
@@ -0,0 +1,21 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import QrBarcodeReader
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
+Intent = autoclass('android.content.Intent')
+Uri = autoclass('android.net.Uri')
+
+class AndroidQrBarcodeReader(QrBarcodeReader):
+
+    def _scan(self):
+
+        intent = Intent()
+        intent.setAction(Intent.ACTION_VIEW)
+        intent.setData(Uri.parse('http://www.onlinebarcodereader.com'))
```

Comment:
Yep, But anyway, this PR completly depending on a remote website is not good for plyer. Either do a native/library-based implementation, but don't require internet for that or external services :)
+1 for closing and reopen with a new approach.